meta-refkit-extra/doc/bluetoothspeaker.rst

```diff
@@ -0,0 +1,75 @@
+Bluetooth Speaker
+#################
+
+This is a demo to convert Intel 570x development board into stand
+alone bluetooth speaker. Main application is a python file
+``btspeaker.py`` and it is run under a Linux user called
+``btspeaker``. The demo uses fairly "stock" components like
+``PulseAudio``, ``BlueZ`` and ``ConnMan``. These are included in the
+IoT Reference OS Kit's gateway profile. The 570x doesn't have audio
+codec or speaker included so if you want to hear actual sound you need
```

Comment:
Is it the lack of codec or the lack of an audio jack connector that is the main problem here? From a user perspective, it feels more intuitive to read that the problem is the lack of an audio jack connector or built-in speaker that forces us to connect via an external device such as a USB audio card of HDMI. By the way, would a standard HDMI TV or screen with audio work (or do you really need a home theater amplifier)?